src/utils/funksjonerForAmplitudeLogging.ts

```diff
@@ -72,7 +72,7 @@ export const useLoggBedriftValgtOgTilganger = (org: OrganisasjonInfo | undefined
         if (isLoading) return;
 
         const navtjenestetilganger = Object.entries(org.altinntilgang)
-            .filter(([key, value]) => key in NAVtjenesteId && value === true)
```

Comment:
dette burde jeg ha fanget tidligere. copilot som foreslo `in` sjekk på et array og så gikk jeg fem på 🤦